spec/stat_tracker_spec.rb

```diff
@@ -46,6 +46,28 @@
     expect(first_game_teams.team_id).to eq("3")
   end
 
+  it "can create a hash with seasons and games" do
+    expect(@stat_tracker.create_season_games_hash).to be_a Hash
+    expect(@stat_tracker.create_season_games_hash.keys.first).to eq("20122013")
+  end
+
+  it "can create seasons in an array" do
+
+    game_path = "./data/games_sampl.csv"
+    teams_path = "./data/teams_sampl.csv"
+    game_teams_path = "./data/game_teams_sampl.csv"
+    locations = {
+      games: game_path,
+      teams: teams_path,
+      game_teams: game_teams_path
+    }
```

Comment:
Since all of the paths/locations are defined in the before block, you could slap an @ on the ones you need instead of defining them all again for each test